pages/index.tsx

```diff
@@ -13,9 +13,9 @@ export default function Home() {
       </Head>
 
       <main className={styles.main}>
-          <Logo size="big"></Logo>
-          <LoginButton></LoginButton>
-          <SkipLoginButton></SkipLoginButton>
+        <Logo size="big"></Logo>
+        <LoginButton></LoginButton>
+        <SkipLoginButton></SkipLoginButton>
```

Comment:
You could write it shorter like this, no need for closing the Buttons like you do, if they don't have child elements:
`<Logo />
<LoginButton />
<SkipLoginButton />`